### DISCUSSION TITLE
Adding ImgBB.com as hosting provider

### DISCUSSION THREAD

**Discussion by o-Oby** [flameshot-org/flameshot#3161] [Q&A]
Hi,

I would like to add imgbb.com as an hosting provider in Flameshot. I inspected the code but I can't find which file where the code for imgur is included... Can someone please guide me ? That would be very helpful.

Thank you very much for your help! :-)

Oby

**Comment by mmahmoudian** [comment]
Searching among open and closed issues can really save your time. It has been discussed and decided that we such features be implemented as plugins.

But if you want to have your own fork of Flameshot and add imgbb, use ripgrep on the source code to quickly find what you are looking for. Logically all the code should be here:

https://github.com/flameshot-org/flameshot/tree/master/src%2Ftools%2Fimgupload
